patches/minecraft/net/minecraft/block/BlockChorusFlower.java.patch

```diff
@@ -0,0 +1,27 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockChorusFlower.java
++++ ../src-work/minecraft/net/minecraft/block/BlockChorusFlower.java
+@@ -51,9 +51,13 @@
+             if (p_180650_1_.func_175623_d(blockpos) && blockpos.func_177956_o() < 256)
+             {
+                 int i = ((Integer)p_180650_3_.func_177229_b(field_185607_a)).intValue();
+-
+-                if (i < 5 && p_180650_4_.nextInt(1) == 0)
++                
++                if (i < 5)
```

Comment:
First off, spacing.
Also, a 'ForgeHooks.cropsPre(x,y,z, rand.nextInt(1)==0)' would make this code and a lot of other patches A LOT cleaner.